app-admin/sudo/Manifest

```diff
@@ -1,3 +1,5 @@
 DIST sudo-1.8.20p2.tar.gz 2930769 BLAKE2B 90338e7d436683f94a9e13e9f0da668f2bbbf2184530feb63166186309881b737b783d2be80f935e5d64fd2a4264928423d2b754121be5caf07acbacbe2a6d42 SHA512 8bf67e687f7a84605fdef8d547b5cd661141b6c8fd25820c33c7e37e97ca7f21f564c3bae691f8a8cd08df7d80338e36a8f06bb5086cc104509d71d6ab1bceda
 DIST sudo-1.8.22.tar.gz 3029051 BLAKE2B c77e05b6e9cee738902d6289327fb5d34d19833d96597f983d8af01434d224dd698f9257b0965a0e480e8d19eb38eef0c8216942ca5217c3fe7516cdf397f7b7 SHA512 5ce10a9302d25bb726e347499d26a0b3697446cfcdf0fd9094ee35198db7b023d5250a53fdcb4184d1a09f5fd2a78fc645bc8e80f265666b05a91f62f49b0695
 DIST sudo-1.8.23.tar.gz 3150674 BLAKE2B 11b1c7bfa372005cda8baf651c4662f6fd15e94ca77f7705b23ca6573424796d5c1f8e47e2874c4b54017141d01a632885ac60c92346d932537048373cad0ede SHA512 a9d61850a4857bfd075547a13efb13b054e4736e3ebe3c8a98a90a090b1d9b9688354ec9725fc99d1d256999b6f9c6ae6215ce9770fcdebd7f24731107b48342
+DIST sudo-1.8.25p1.tar.gz 3189951 BLAKE2B ebfedaad62e60f625db8c46a5c8f19977a5ec0a86bab3b34d91096c08e8b8ece056ba312f9fecd4cdd704fc17d49a36681b41cd40269df7c67cd66d80c0d8efb SHA512 b1445be688d3c1dd7efbdfab68977a7a9b6fd6887191dc99ca717117eec0a550492642556cd55ca5873d054ddc5ccc2b87b2c34602e1ffc729ab6fbc4e523a72
+EBUILD sudo-1.8.25_p1-r1.ebuild 6530 BLAKE2B 522ae455b21dc5d1435826a518790fb1bacfa43a6edd958c8a9c0d0e2ef9dd2588f82ff4b9b8f89801b7e24b622aa84c72e3a173110c8724edf11bcdce9385fd SHA512 10f03f5986fcab0087bb4e0e968b231bc13a7a2c0af14935fb4cf21ae67e8952dc7338dfe90f9ac38bc89cad12978ac259fe2012fd0023214df5b1106f1ea73e
```

Comment:
Please remove the digests for `EBUILD`. There's no need for it, as we use [`thin-manifests`](https://github.com/gentoo/libressl/blob/master/metadata/layout.conf#L2).